src/provider.rs

```diff
@@ -908,3 +1055,23 @@ mod tests {
         Ok(())
     }
 }
+
+/// Create a [`quinn::ServerConfig`] with the given keypair and limits.
+pub fn make_server_config(
+    keypair: &Keypair,
+    max_streams: u64,
+    max_connections: u32,
+    alpn_protocols: Vec<Vec<u8>>,
+) -> anyhow::Result<quinn::ServerConfig> {
```

Comment:
I'm wondering whether to ponder a builder here too.  But I'm not sure.

Reply:
It's a pretty straightforward fn. Just because it has 2 numeric args?

Reply:
Came here for the same issue, it's pretty odd to read at the call site, there are just two random number constants, with no clarity what they mean. Could either make this a builder (or just a simple options struct) or at least name the numbers at the call sites as constants.

Reply:
I guess I don't feel the pain as much because of vscode

![image](https://user-images.githubusercontent.com/248257/222095411-8e4cb03c-52b8-4b48-bf01-f124c39c34b5.png)

Will put it in constants.